fun.py

```diff
@@ -215,7 +215,7 @@ def pizza(self):
 
     def fade(self):
         while True:
-            bright = 255
+            bright = random.randint(0,255)
```

Comment:
I recommend starting with a random (but nice) predetermined color from the patterns.py file.  If you just set the three randomly, you will get some shades of color, but you're destined to just get a lot of ugly off-white shades frequently